ode/unit_test/Test_ODE_Newton.hpp

```diff
@@ -49,16 +49,23 @@ struct NewtonSolve_wrapper {
   void operator()(const int idx) const {
     // Take subviews to create the local problem
     auto local_x = Kokkos::subview(
-        x, Kokkos::pair(my_nls.neqs * idx, my_nls.neqs * (idx + 1)));
+        x, Kokkos::pair<decltype(my_nls.neqs), decltype(my_nls.neqs)>(
```

Comment:
I am not very happy with `decltype(my_nls.neqs)`, can we just assume `int` here and potentially static_cast the input parameter instead?

Reply:
@lucbv sounds good, I'll update the PR with these changes